src/Umbraco.Headless.Client.Net/Security/AuthenticationService.cs

```diff
@@ -14,30 +14,34 @@ public class AuthenticationService : IAuthenticationService
         private HttpClient _apiHttpClient;
         private UserOAuthEndpoints _apiEndpoints;
         private MemberOAuthEndpoints _cdnEndpoints;
+        private readonly RefitSettings _refitSettings;
 
-        public AuthenticationService(string projectAlias, HttpClient cdnHttpClient = null,
-            HttpClient apiHttpClient = null) : this(new HeadlessConfiguration(projectAlias), cdnHttpClient, apiHttpClient)
+        public AuthenticationService(string projectAlias, RefitSettings refitSettings, HttpClient cdnHttpClient = null,
```

Comment:
This is a breaking change, I think we should create overloads for the constructors instead and maybe mark the old one as obsolete

Reply:
looks like you can't really use the `AuthenticationService` from the outside, so this might be fine :)